Comparator.lib

```diff
@@ -123,7 +123,9 @@ ALIAS LM393
 $FPLIST
  SOIC*3.9x4.9mm*P1.27mm*
  DIP*W7.62mm*
- TO?99*
+ SOP*5.28x5.23mm*P1.27mm*
+ VSSOP*3.0x3.0mm*P0.65mm*
+ TSSOP*4.4x3mm*P0.65mm*
 $ENDFPLIST
```

Comment:
WSON is missing here. But I assume you left it out intentional because the EP is not in the symbol. The datasheet says that EP needs to be connected to GND.

Reply:
Correct.